apps/fluent-tester/src/E2E/Avatar/specs/Avatar.spec.win.ts

```diff
@@ -22,3 +32,25 @@ describe('Avatar Testing Initialization', function () {
     expect(AvatarPageObject.didAssertPopup()).toBeFalsy(AvatarPageObject.ERRORMESSAGE_ASSERT);
   });
 });
+
+describe('Avatar Accessibility Testing', () => {
+  beforeEach(() => {
+    AvatarPageObject.scrollToTestElement();
```

Comment:
I would move this beforeEach block into each test to see if it's causing the issue.